pom.xml

```diff
@@ -171,6 +188,7 @@
     <allowscript.javadoc.qualifier></allowscript.javadoc.qualifier>
     <!-- Invalid flag for old javadoc versions (default to empty). -->
     <doclint.javadoc.qualifier></doclint.javadoc.qualifier>
+    <numbers.junit.bom.version>5.4.2</numbers.junit.bom.version>
```

Comment:
I would move this to the location of other versions, e.g. below numbers.mathjax.version